src/anomalib/data/ucsd_ped.py

```diff
@@ -216,13 +216,13 @@ class UCSDped(AnomalibVideoDataModule):
 
     def __init__(
         self,
-        root: Path | str,
-        category: str,
+        root: Path | str = "./datasets/uscd",
+        category: str = "UCSDped1",
```

Comment:
Can we make UCSDped2 the default? UCSDped1 does not have a ground truth annotation for every video